.github/workflows/release.yml

```
@@ -17,20 +17,26 @@ jobs:
     - name: Install msys2 dependencies
       shell: msys2_shell.cmd {0}
       run: |
-          -defterm -no-start -here -mingw64 -c 'pacman -S --noconfirm libxslt'
-
+        -defterm -no-start -here -mingw64 -c 'pacman -S --noconfirm libxslt'
     - name: Build Generic64
+      shell: msys2_shell.cmd {0}
       run: |
-        c:\msys64\msys2_shell.cmd -defterm -no-start -here -mingw64 -c "CFLAGS='-O3 -g0 -march=ivybridge -mtune=native' ASMFLAGS='-march=ivybridge -mtune=native' make generic64/libXKCP.so"
+        -defterm -no-start -here -mingw64 -c "make generic64/libXKCP.so"
```

Maybe try this...
```suggestion
      shell: msys2_shell.cmd -defterm -no-start -here -mingw64 {0}
      run: |
         make generic64/libXKCP.so
```
I think it's going to replace `{0}` with a file it writes the script in `run:` to, so drop the `-c`.